src/main/java/team4/footwithme/team/service/TeamServiceImpl.java

```diff
@@ -52,6 +53,9 @@ public TeamDefaultResponse createTeam(TeamDefaultServiceRequest dto) {
         );
         Team createdTeam = teamRepository.save(entity);
 
+        //TODO :: 현재 유저 팀장으로 TeamMember에 저장
+        teamMemberRepository.save(TeamMember.create(createdTeam, member, TeamMemberRole.CREATOR));
```

Comment:
개인적인 생각인데 CREATOR, MEMBER 이렇게 역할마다 다른 create가 생긴다면 정적 팩토리 메서드를 추가해서 좀 더 명확하게 표현하는건 어떨까요 ?

```suggestion
        teamMemberRepository.save(TeamMember.createCreator(createdTeam, member));
```

이렇게 하면 더 표현이 명확해 질 것 같아요 !

Reply:
오 좋은것 같습니다! 메서드를 추가하는게 더 명확한 것 같아요!
적용해보겠습니다!